### PR TITLE
T/tp2322 Various docs fixes

### PR DESCRIPTION
**Pull request by Comandeer**
This PR fixes following issues with docs:

* things leaking into global scope,
* links to non-existent classes and methods,
* incorrect or duplicated variables/parameters names.

### CODE REVIEW

File: plugins/tabletools/plugin.js [hunk 1]
```diff
@@ -1896,6 +1896,7 @@
 	 *
 	 * @since 4.7.0
 	 * @cfg {Boolean} [tableImprovements=true]
+	 * @member CKEDITOR.config
```
Comment (mlewand) [inline plugins/tabletools/plugin.js]: Please revert this change as it will be deprecated by a soon to be merged table selection plugin extraction.

Comment (mlewand) [inline plugins/tabletools/plugin.js]: Just to avoid conflicts.

**Comment by Comandeer** [comment]
Ok, reverted change.
